CONTRIBUTING.md

```diff
@@ -11,36 +11,34 @@ Make sure you have these prerequisites working:
 * The C++ compiler for your platform (invokable as `gcc`)
 * Optional: `latex` for building the pdf documentation
 * Optional: `kindlegen` for building the Kindle documentation
-* If you're running OSX, make sure you have some version of `libevent` installed
+* If you're running OSX, make sure you have some version of `libevent` installed (needed for Ddox)
 
 ## Getting the code
 
-Create a working directory for the D language, e.g. `~/code/d`. The remainder
-of this document calls that directory henceforth `$R` from "Root". To get the
-code:
+If you already have a [working directory for the D language](https://wiki.dlang.org/Building_under_Posix#Fetch_repositories_from_GitHub),
+e.g. `~/dlang`, change to it.
```

Comment:
Not sure that it is a good idea to suggest the same working directory as the one install.sh manages.

Reply:
It use ~/dlang for a long time now and since we moved the checkouts into `.generated`, I haven't had any problems. Though it would have been nice for the install.sh script to support an alternate directory - see https://github.com/dlang/installer/pull/274

Reply:
> It use ~/dlang for a long time now

and I recall you said it caused problems due to conflicts between what the installer tool does and what the makefiles did.

I agree, we should change this, probably in the documentation.

Reply:
Sorry for the typo: s/It/I/

> and I recall you said it caused problems due to conflicts between what the installer tool does and what the makefiles did.

Yep, it used to. However, we fixed that with https://github.com/dlang/dlang.org/pull/1889

> I agree, we should change this, probably in the documentation.

This was an outdated comment I saw by chance and accidentally never replied, so I thought it's a good idea to leave an answer here, but I don't see any need for action anymore.

Reply:
> Yep, it used to. However, we fixed that with #1889

That's just one problem we know about.

> This was an outdated comment I saw by chance and accidentally never replied, so I thought it's a good idea to leave an answer here, but I don't see any need for action anymore.

This is a minor issue, but otherwise I disagree. We have to places in the official material that refer to the same directory for different purposes. They don't conflict *now*, but it's still an unnecessary source of confusion and potential problems, for the same reason you don't use one variable for two different purposes.